src/error.rs

```diff
@@ -58,9 +58,9 @@ pub enum DecodeError {
 
 #[derive(Error, Debug)]
 pub enum ServiceError {
-    #[error("service {0}")]
+    #[error("service: {0:?}")]
```

Comment:
why the gratuitous debug for errors? I actually have cleaner messages for some of these?